app/controllers/api/v1x0/portfolios_controller.rb

```diff
@@ -102,6 +103,21 @@ def portfolio_params
       def portfolio_copy_params
         params.permit(:portfolio_id, :portfolio_name)
       end
+
+      def permissions
+        params.require(:permissions).tap {|perm| verify_permissions(perm) }
```

Comment:
This reads a little rough to me, would you be open to changing it to something more like:
`verify_permissions(params.require(:permissions))`

Since the `verify_permissions` method takes an array and the last statement is `.each` it should return the array. We could just pass it the array correct?

Reply:
@lindgrenj6 The verify_permission is not sanitizing the data and sending it back, it is sort of a function that raises error if any of the values is incorrect. so I could do it in 3 steps
``
def permissions
   perm = params.require(:permission)
   verify_permission(perm)
   perm
end
```

The tap just makes it a one liner